### DISCUSSION TITLE
Github is not the place for facebook-like feeds

### DISCUSSION THREAD

**Discussion by pbasov** [community/community#13433] [Other Feature Feedback, Questions, & Ideas]
Why does every service on the internet needs to degrade into trying to feed my data into a recommendation system?

Remove this, before you go too far and add stories.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
